ui/component/fileRenderFloating/view.jsx

```diff
@@ -398,3 +443,176 @@ export default function FileRenderFloating(props: Props) {
     </Draggable>
   );
 }
+
+type GlobalStylesProps = {
+  videoAspectRatio: number,
+  videoTheaterMode: boolean,
+  appDrawerOpen: boolean,
+  initialPlayerHeight: ElementRef<any>,
+  isFloating: boolean,
+  fileViewerRect: any,
+  mainFilePlaying: boolean,
+  isLandscapeRotated: boolean,
+};
+
+const PlayerGlobalStyles = (props: GlobalStylesProps) => {
+  const {
+    videoAspectRatio,
+    videoTheaterMode,
+    appDrawerOpen,
+    initialPlayerHeight,
+    isFloating,
+    fileViewerRect,
+    mainFilePlaying,
+    isLandscapeRotated,
+  } = props;
+
+  const isMobile = useIsMobile();
+  const isMobilePlayer = isMobile && !isFloating; // to avoid miniplayer -> file page only
+
+  const heightForViewer = getPossiblePlayerHeight(videoAspectRatio * fileViewerRect.width, isMobile);
+  const widthForViewer = heightForViewer / videoAspectRatio;
+  const maxLandscapeHeight = getMaxLandscapeHeight(isMobile ? undefined : widthForViewer);
+  const heightResult = appDrawerOpen ? `${maxLandscapeHeight}px` : `${heightForViewer}px`;
+  const amountNeededToCenter = getAmountNeededToCenterVideo(heightForViewer, maxLandscapeHeight);
+
+  // forceDefaults = no styles should be applied to any of these conditions
+  // !mainFilePlaying = embeds on markdown (comments or posts)
+  const forceDefaults = !mainFilePlaying || videoTheaterMode || isFloating || isMobile;
```

Comment:
`Optional`
This commit `Add another comment` can probably be squashed into `Fix miniplayer size issues` now that review is done.